specification/ai/Azure.AI.Projects/red-teams/models.tsp

```diff
@@ -120,6 +120,17 @@ model AzureOpenAIModelConfiguration extends TargetConfig {
   modelDeploymentName: string;
 }
 
+@doc("Azure Foundry Agent configuration. The API version would be selected by the service for querying the model.")
+@added(Versions.v2025_05_15_preview)
+@removed(Versions.v1)
+model AzureFoundryAgentConfiguration extends TargetConfig {
```

Comment:
We don't use Product/Marketing names in class names. I would not use "Foundry". Name it something that indicates what it's used for (e.g. "RedTeamAgentConfiguration")